app/scripts/migrations/088.ts

```diff
@@ -252,12 +252,6 @@ function migrateData(state: Record<string, unknown>): void {
         allDetectedTokens,
         (_, chainId: string) => toHex(chainId),
       );
-    } else if (hasProperty(tokensControllerState, 'allDetectedTokens')) {
-      global.sentry?.captureException?.(
```

Comment:
I think this is the wrong error. The one in the Sentry report is on line 167

Reply:
Absolutely, Mark, thanks for pointing it out